Hogajama/hogajama-domain/src/main/java/com/gepardec/hogarama/domain/sensor/SensorNormalizer.java

```diff
@@ -13,6 +29,12 @@ public SensorData normalize(SensorData data) {
 	}
 
 	private MappingType mappingTypeOf(SensorData data) {
+	    Optional<Sensor> sensor = sensors.getByDeviceId(data.getSensorName());
```

Comment:
`data.getSensorName()`  corresponds the current sensor data format? We will deinitely introduce some abstraction on this, as long as we will have to support both formats simultaneously.

Reply:
We decided that we use the SensorData.name in the old Data-Format as device-ID. That the code that does that.